src/user/user.c

```diff
@@ -13,12 +13,19 @@
  * limitations under the License.
 */
 
-#include <linux/bpf.h>
+#include <bpf/libbpf.h>
 #include "exec.skel.h"
 #include <unistd.h>
+#include <bpf/bpf.h>
 
 int main(int ac, char **argv)
 {
+        int cpus = libbpf_num_possible_cpus(); // the number of CPUs on your machine
+        unsigned long long values[cpus]; // number of executed processes on each CPU
```

Comment:
Usually, if a variable name is well chosen, it does not need a comment. (which is helpful because I might start reading the code later in the file and stumble upon this "value" and I would be really unsure what it is, that would force me to go back and read the comment)
This is just a proposal but executions_nb_on_cpu does not need a comment and when you use it `executions_nb_on_cpu[3]` it reads more easily than `value[3]`

Reply:
Similar things can be said for other variables, maybe cpus_nb makes more sense than cpus, because it avoids one comment too.

Reply:
This is actually part of the linux style guide https://www.kernel.org/doc/html/v4.10/process/coding-style.html#commenting over-commenting code does not help. Also a good comment describes what the function does, not how.

One good reason for this is that the "what" shouldn't change over time, but the "how" might change and most likely your comment will be left outdated (cf your "loop over the values, add them up and reset the value of each CPU" which was already outdated before you even submitted the PR :) )

Another good reason is that the reader is expected to know C. I expect anyone who glances at:
```
for (int i = 0; i < cpus; i++) {
        counter += values[i];
}
```
To figure out that this "loop over the values and add them up" in a split of a second. What might help the reader though is what are those "values", and why are they being added up? Maybe something like "Accumulate execution number on each CPU" is already more helpful. And maybe someday the implementation will change to something else but the comment will stay relevant.